01_fib/hsy3418/main_test.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := strconv.Quote("1\n1\n2\n3\n5\n8\n13\n")
+	actual := strconv.Quote(buf.String())
+
+	if expected != actual {
+		t.Errorf("Unexpected output in main()")
```

Comment:
Provide more descriptive output for failure. You can print expected and actual outputs so that reason can be identified looking at test output if test failed.
```
t.Errorf("Unexpected output in main()\nexpected: %q\nactual: %q", expected, actual)
```